fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -39,6 +46,13 @@ def self.available_options
                                        verify_block: proc do |value|
                                          UI.user_error!("Please pass a valid command. Use one of the following: #{available_commands.join(', ')}") unless available_commands.include? value
                                        end),
+          FastlaneCore::ConfigItem.new(key: :dependencies,
+                                       description: "Carthage dependencies to update",
+                                       default_value: [],
+                                       is_string: false,
```

Comment:
Please add `type: Array,` to the params for this `ConfigItem`. That should remove the need to manually verify the data type in the `verify_block` 👍